spec/api/v1/sg_email_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+# rubocop:disable Layout/AlignHash
+
+require 'rails_helper'
+
+RSpec.describe 'SendGrid API', type: :request do
+  describe 'POST /api/v1/sg_emails' do
```

Comment:
Do we have this endpoint? Shouldn't we have only one endpoint for the whole application ?

Reply:
Let's clean up the tests in the next feature branch, okay?